04-oop-basic-intro-to-dom/1-column-chart/index.js

```diff
@@ -63,9 +63,14 @@ export default class ColumnChart {
     return element;
   }
 
-  destroy() {}
+  destroy() {
+    this.remove();
+  }
 
-  update() {}
+  update(newData) {
+    this.data = newData;
+    this.element = this.createElement();
```

Comment:
this.element.querySelector("[data-element="body"]).innerHtml = this.createColumns();